Firmware/first_lay_cal.cpp

```diff
@@ -212,3 +212,40 @@ void lay1cal_square(char *cmd_buffer, uint8_t i)
     sprintf_P(cmd_buffer, fmt2, (35 - (i + 1)*width * 2), extr_short_segment);
     enquecommand(cmd_buffer);
 }
+
+
+//! @brief Set the flow in order to the installed nozzle
+//!
+//! This function must be called before to start to print in order to 
+//! enqueue the correct flow depending the nozzle size stored on the 
+//! eeprom
+//!
+void lay1cal_set_flow()
+{    
+   uint8_t nDiameter;
```

Comment:
I made those changes, as you said I removed the "default" case but I had to add the case for "ClNozzleDiameter::_Diameter_Undef:" in order to avoid the warning, so now if there is any new nozzle size there will be a warning at compilation time.

```
case ClNozzleDiameter::_Diameter_Undef:
        enquecommand_P(PSTR("M221 S100"));
        break;
```
Now is 4 bytes expensive but it worth the extra bytes because now use the defined values on the enum class ClNozzleDiameter.